front/lib/data_sources.ts

```diff
@@ -58,17 +57,3 @@ export function isWebsite(
 export function isManaged(ds: DataSource): ds is DataSource & WithConnector {
   return ds.connectorProvider !== null && !isWebsite(ds);
 }
-
-const STRUCTURED_DATA_SOURCES: ConnectorProvider[] = [
-  "google_drive",
-  "notion",
-  "microsoft",
-];
-
-export function canContainStructuredData(ds: DataSource): boolean {
```

Comment:
I think we need this feature to avoid showing datasource that can't contains tables in DataViewsSelector when viewType == table.

Reply:
(a bug I just saw)

Reply:
Let me take a look.